.github/workflows/ci.yaml

```diff
@@ -21,19 +21,53 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
-  lint-live:
-    name: Lint / Live
+  # lint-oss tries to run this action against open source repositories not in
+  # open-turo to make sure this is usable outside the org.
+  # Repositories added to this list should have pre-commit hooks that don't
+  # require external tooling to be installed.
+  # Repositories added to this list must have passing pre-commit hooks.
+  lint-oss:
+    name: Lint / Open source
     runs-on: ubuntu-latest
     strategy:
       matrix:
         repos:
-          - prowler-cloud/prowler
+          - disposable/disposable
+          - cloud-custodian/cloud-custodian
+          - aws/aws-sdk-pandas
+          - google/uv-metrics
     steps:
       - uses: actions/checkout@v4
         with:
           repository: ${{ matrix.repos }}
-      - uses: open-turo/actions-python/lint@v1
+      # TODO: Remove branch reference
+      - uses: open-turo/actions-python/lint@live-tests
         with:
           checkout-repo: false
         # with:
         #   github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  # test-oss tries to run this action against open source repositories not in
+  # open-turo to make sure this is usable outside the org.
+  # Repositories added to this list should be pyproject.toml based repositories.
+  # Repositories added to this list must have passing builds.
+  test-oss:
+    name: Test / Open source
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        repos:
+          - aws/aws-sdk-pandas
+          - copier-org/copier
+          - hvac/hvac
+          - tiangolo/asyncer
+          - tiangolo/fastapi
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: ${{ matrix.repos }}
+      # TODO: Remove branch reference
+      - uses: open-turo/actions-python/test@live-tests
+        with:
+          checkout-repo: false
```

Comment:
Neat! 🚀 

qq: do we want to test also things like making sure that the correct python version is being installed or something along these lines or we just want to make sure the action runs?